content/post/agile-analytics-p2.md

```diff
@@ -0,0 +1,56 @@
+
+When you hear something like "Ooh shoot. I noticed that issue before but I was so focused on finishing my story before the sprint ended that I didn't look closer" then you may be suffering from this. This "illusion of effectiveness", where the team feels like they're getting all their stories done (collecting those sweet, sweet story points) is pernicious because it *feels* productive despite missing out on potential high-value opportunities. The analytics team is *supposed* to be generating novel insights, which means that there has to be some tolerance for unplanned investigations.
+
+As a researcher, I've often found that the most valuable insights I've generated have come from investigating something that looked odd or interesting and was only tangentially related to the task at hand. These sorts of exploratory analysis make up the "science" part of data science and we want to ensure that our analysts have the time and the inclination to pursue these opportunities and generate novel insights.
+
+## Exploratory data analysis
+
+Exploratory data analysis is an important part of the "basic research" function of an analytics or data science team. Unfortunately, it's really hard to write "user stories" or "acceptance criteria" for this sort of work because ... the whole point is that you aren't sure what you're going to find, and it requires a researcher's intuition to know when the story is "done" (because there's always more things you *could* look into, but often those marginal investigations aren't valuable).
+
+The nature of exploratory data analysis means that the objectives of the analysis may change *as you do the work*. You sometimes realize "oh x isn't actually very interesting -- the real issue is understanding y." If your scrum process is too inflexible, you can end up expending effort against tasks that aren't valuable simply because they're required to get the story done.  While (software) engineering is a discipline of creatively applying mathematical tools to a well-defined problem, data science is about creatively applying mathematical tools to *poorly defined* problems, and it's this fundumental difference in nature that makes partitioning the work into small well-scoped tasks sometimes impossible for data science.
+
+## Product ownership / story-writing
+
+When you're trying to build a model of customer churn, there's some amount of judgement and art that goes into doing good data science. The acceptance criteria of "predict which customers are going to churn" is too vague to be helpful and doesn't capture the art of investigating a problem by poking at it from multiple sides.  Additionally, it can be very difficult for someone external to write stories or be the product owner for the team. Often, the data scientists or analysts are *the* domain experts in the company and there isn't anyone else who is better informed about what the next step in a research program should be. 
```

Comment:
I think this is related to the fact that some work is likely to be generated by the data team - such as a churn model. There likely is a stakeholder somewhere in the org that cares a lot about churn and may do something about it if they had a model that answers their questions about churn. But if they didn't generate the original ask, how is this information surfaced to them? And should they be involved right away instead of post hoc? And if so, whose job is it to find them and facilitate the project ideation?